src/main/java/uk/gov/hmcts/reform/pdf/service/client/S2sClient.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.pdf.service.client;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@FeignClient(name = "s2s", url = "${s2s.url}")
```

Comment:
pdf-service shouldn't need its own client, there's a common one being built here.
https://github.com/hmcts/service-authorisation-access-client

Reply:
exactly _is being built_. not ready, not out there for grabs

Reply:
would be nice to unify... can be done later of course

Reply:
Looks like the current effort (and first release) is focused on retrieving the token.
I'd say use this for now, migrate to java client lib once it's ready.